### PR TITLE
BCDA-7971: Update curl commands with double quotes

### PR DESCRIPTION
**Pull request by austincanada**
This feature updates all curl command snippets that have single quotes to double quotes in order for them to run properly on Windows and UNIX based systems.

## 🎫 Ticket

https://jira.cms.gov/browse/BCDA-7971

## 🛠 Changes

Updated the `_includes/` files that contained cURL snippets.

## ℹ️ Context for reviewers

N/A

## ✅ Acceptance Validation

Changes verified on local static site.

## 🔒 Security Implications

- [ ] This PR adds a new software dependency or dependencies.
- [ ] This PR modifies or invalidates one or more of our security controls.
- [ ] This PR stores or transmits data that was not stored or transmitted before.
- [ ] This PR requires additional review of its security implications for other reasons.

If any security implications apply, add Jason Ashbaugh (GitHub username: StewGoin) as a reviewer and do not merge this PR without his approval.

### CODE REVIEW

File: _includes/guide/bcda_v2.html [hunk 1]
```diff
@@ -20,14 +20,14 @@ <h2>
         </tr>
         <tr>
             <td>
-                <pre><code>curl -X GET 'https://sandbox.bcda.cms.gov/api/v2/Group/all/$export' \
+                <pre><code>curl -X GET "https://sandbox.bcda.cms.gov/api/v2/Group/all/$export" \
```
Comment (greg-pf) [inline _includes/guide/bcda_v2.html:23]: Maybe we can sneak in highlighting of `v2`?
```suggestion
                <pre><code>curl -X GET "https://sandbox.bcda.cms.gov/api/<span style="background: yellow">v2</span>/Group/all/$export" \
```

File: _includes/build/bcda_v2.html [hunk 2]
```diff
@@ -20,14 +20,14 @@ <h2>
             </tr>
             <tr>
                 <td>
-                    <pre><code>curl -X GET 'https://api.bcda.cms.gov/api/v2/Group/all/$export' \
+                    <pre><code>curl -X GET "https://api.bcda.cms.gov/api/v2/Group/all/$export" \
```
Comment (greg-pf) [inline _includes/build/bcda_v2.html:23]: Same comment on highlighting `v2`

**Review by greg-pf — APPROVED** [review]
Looks good to me. I left an optional/not blocking suggestion for highlighting `v2` based on your friction log the other week, but I'm not sure if that <span> tag will pass accessibility scans

File: _includes/guide/bcda_v2.html (hunk 1 above)
Comment (austincanada) [reply _includes/guide/bcda_v2.html:23]: We can add those to either one of the static site tickets for next sprint, if there needs to be accessibility checks for it.

**Pull request merged**